Add cost formula and limitations (#313)
seqeralabs/nf-tower-docs: docs/monitoring/aggregate_stats.md

### Estimated cost

> The estimated cost is an estimate of the total compute cost of all tasks in the workflow run. The compute cost of a task is calculated as follows:

Reviewer: ```suggestion
The estimated cost estimates the total compute cost of all tasks in the workflow run. The compute cost of a task is calculated as follows:
```

\( \text{Task cost} = \text{VM hourly rate} \times \text{VM fraction} \times \text{Task runtime} \)

\( \quad \text{VM fraction} = \text{max} ( \frac{\text{Task CPUs}}{\text{VM CPUs}}, \frac{\text{Task memory}}{\text{VM memory}} ) \)

\( \quad \text{Task runtime} = ( \text{Task complete} - \text{Task start} ) \)

Tower uses a database of prices for AWS and Google Cloud, accross all instance types, regions, and zones, to fetch the VM price for each task. This database is updated periodically to reflect the most recent prices.

!!! note
    Prior to Tower Enterprise 22.4.x, the cost estimate used `realtime` instead of `complete` and `start` to measure the task runtime. The `realtime` metric tends to underestimate the billable runtime because it doesn't include the time required to stage input and output files.

> This estimate does not account for the cost of storage, network, the head job, or the way in which tasks are mapped to VMs. As a result, the estimated cost tends to underestimate the true cost of a workflow run, so it should only be used as a heuristic. You should use the cost reporting tools for your cloud provider (such as AWS Cost Explorer) for accurate cost accounting. You can use [Resource labels](../resource-labels/overview.md) in your compute environments to annotate and track the actual cloud resources that are consumed by a workflow run.

Reviewer: ```suggestion
This estimate does not account for the cost of storage, network, the head job, or how tasks are mapped to VMs. As such, the estimated cost tends to underestimate the true cost of a workflow run and should only be used as a heuristic. For accurate cost accounting, you should use the cost reporting tools for your cloud provider (such as AWS Cost Explorer). You can use [Resource labels](../resource-labels/overview.md) in your compute environments to annotate and track the actual cloud resources consumed by a workflow run.
```
